features/interpolate-size.yml

```
@@ -0,0 +1,10 @@
+name: interpolate-size
+description: The `interpolate-size` CSS property allows animations and transitions to interpolate between a numeric value and keyword values such as `auto` or `fit-content`.
```

Since you can explicitly _not_ allow this behavior, I think we have to say that this thing is where the author chooses a behavior:

```suggestion
description: The `interpolate-size` CSS property sets whether animations and transitions interpolate between a numeric value and a keyword value, such as from a fixed length to `auto` or `fit-content`.
```